gobblin-runtime/src/main/java/org/apache/gobblin/runtime/spec_store/FSSpecStore.java

```diff
@@ -258,7 +258,7 @@ protected Spec readSpecFromFile(Path path) throws IOException {
     Spec spec = null;
 
     try (FSDataInputStream fis = fs.open(path);) {
-      spec = this.specSerDe.deserialize(IOUtils.toByteArray(fis));
+      spec = this.specSerDe.deserialize(ByteStreams.toByteArray(fis));
```

Comment:
This is due to toByteArray() being removed in v1.4 of apache-commons-codec, so I switched to the equivalent Guava method to avoid issues in the IDE with method not found errors